lib/analyze-commits.js

```diff
@@ -0,0 +1,72 @@
+const { join } = require("path");
+const fetch = require("node-fetch");
+const eik = require("@eik/cli");
+const common = require("@eik/common");
+
+const messages = {
+  IVERSIONUPDATE: "Version: ✅ Updated Eik version from v%s to v%s.",
```

Comment:
Perhaps separating the words would make it easier to read? 

Also any need for the `messages` object? `const I_VERSION_UPDATE = ""` might be better?

Reply:
I like that. Will change.